docs/workflows/manage-content.rst

```diff
@@ -1,7 +1,7 @@
 .. _content-management:
 
-Manage Container Content in a Repository
-========================================
+Manage Content in a Repository
+==============================
```

Comment:
:shrug:

Reply:
The whole point of these changes was to shrink the names of chapters to make them more pleasant for users' eyes. Why would a user need to see again that we are dealing with Container content if it is known from the context?